modules/virt-importing-rhel-image-boot-source-web.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-creating-and-using-boot-sources.adoc
+
+[id="virt-importing-rhel-image-boot-source-web_{context}"]
+= Importing a Red Hat Enterprise Linux image as a boot source
+
+You can import a Red Hat Enterprise Linux (RHEL) image as a boot source by specifying the URL address for the image.
+
+.Prerequisites
+* You must have access to the web server with the operating system image. For example: Red Hat Enterprise Linux web page with images.
+
+.Procedure
+
+. In the {VirtProductName} console, click *Workloads* -> *Virtualization* from the side menu.
+
+. Click the *Templates* tab.
+
+. Identify the RHEL template for which you want to configure a boot source and click *Add source*.
+
+. In the *Add boot source to template* window, select *Import via URL (creates PVC)*  from the *Boot source type* list.
+
+. Click *RHEL download page* to access the Red Hat Customer Portal. A list of available installers and images is displayed on the Download Red Hat Enterprise Linux page.
+
+. Identify the Red Hat Enterprise Linux KVM guest image that you want to download. Right-click *Download Now*, and copy the URL for the image.
+
+. In the *Add boot source to template* window, paste the copied URL of the guest image into the *Import URL* field, and click *Save and import*.
+
+.Verification
+. In the *Templates* tab, verify that the *Boot source* column displays *Available* for the RHEL virtual machine template. You can use this template to create RHEL virtual machines.
```

Comment:
Per our mod docs guidelines, use a bulleted list for single step procedures instead of a numbered list.

Reply:
Updated the content.